client/my-sites/checkout/composite-checkout/components/wp-checkout-order-summary.js

```diff
@@ -113,12 +142,16 @@ function CheckoutSummaryFeaturesList() {
 	const isJetpackNotAtomic = useSelector(
 		( state ) => isJetpackSite( state, siteId ) && ! isAtomicSite( state, siteId )
 	);
+	const { isMonthlyPricingTest = false, hasMonthlyPlan = false } = props;
 
 	let refundText = translate( 'Money back guarantee' );
 	if ( hasDomainsInCart && ! hasPlanInCart ) {
 		refundText = translate( '4 day money back guarantee' );
 	} else if ( hasPlanInCart && ! hasDomainsInCart ) {
 		refundText = translate( '30 day money back guarantee' );
+		if ( isMonthlyPricingTest && hasMonthlyPlan ) {
+			refundText = translate( '7 day money back guarantee' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( '4 day money back guarantee' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/monthly-pricing-abtest)